python/lsst/ctrl/oods/butlerAttendant.py

```diff
@@ -325,7 +263,8 @@ def clean(self):
         for x in ref:
             uri = None
             try:
-                uri = butler.getURI(x, collections=x.run)
+                # uri = butler.getURI(x, collections=x.run)
+                uri = butler.getURI(x)
```

Comment:
Above (around new line 230), I think we want to `queryCollections` for RUN collections based on the `cleanCollections` config.  But maybe that's another ticket.

Reply:
I'll put this request into another ticket.